app/main.py

```diff
@@ -1,8 +1,42 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
```

Comment:
Each argument should start from a new line here, including `self`. Reformat your code here and for `CarWashStation` to meet this.